server/src/main/java/org/spine3/server/CommandBus.java

```diff
@@ -121,11 +117,21 @@ protected CommandBus(CommandStore commandStore) {
      * <p>The command can be posted if it has either dispatcher or handler registered with
      * the command bus.
      *
-     * @param command the command instance to check
+     * <p>The method intended to be used with the instances of command messages extracted from {@link Command}.
+     *
+     * <p>If {@code Command} instance is passed, its message is extracted and validated.
+     *
+     * @param command the command message to check
      * @return the result of {@link Responses#ok()} if the command is supported,
      *         {@link CommandValidation#unsupportedCommand(Message)} otherwise
      */
     public Response validate(Message command) {
+        if (command instanceof Command) {
+            final Command outerCommand = (Command) command;
+            //noinspection AssignmentToMethodParameter
```

Comment:
See my comment on this matter below. But I guess, it's better to avoid the suppression and make the intent clearer.